tests/unit_tests/simpll/DifferentialFunctionComparatorTest.cpp

```diff
@@ -203,20 +215,24 @@ class DifferentialFunctionComparatorTest : public ::testing::Test {
                 FunctionType::get(Type::getVoidTy(CtxL), {}, false),
                 GlobalValue::ExternalLinkage,
                 "F",
-                &ModL);
+                &*ModL);
```

Comment:
```suggestion
                ModL.get());
```
same below (multiple occurrences)

Reply:
Thanks, fixed.